setup.py

```diff
@@ -163,13 +175,15 @@ def run(self):
 class CITest(TestCommand):
     """Run all CI tests."""
 
-    description = 'run all CI tests: unit and doc tests, linter'
+    description = "run all CI tests: unit and doc tests, linter"
 
     def run(self):
         """Run unit tests with coverage, doc tests and linter."""
-        coverage_cmd = 'python3.6 setup.py coverage %s' % self.get_args()
-        lint_cmd = 'python3.6 setup.py lint'
-        cmd = '%s && %s' % (coverage_cmd, lint_cmd)
+        coverage_cmd = f"python3.6 setup.py coverage {0}".format(
```

Comment:
Could you change these `python3.6` to `python3`? I think this `CITest` `ci` command was only used on Travis, but since we're already moving away from `python3.6` let's take the opportunity to update this. 

In the future, we can remove the `CITest` completely, on GitHub actions most Napps are using these commands `envlist = coverage,lint` fyi.